distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Replace_Params.enso

```diff
@@ -0,0 +1,17 @@
+import Standard.Base.Data.Boolean.Boolean
+import Standard.Base.Error.Error
+import Standard.Base.Meta.Type
+import Standard.Base.Nothing.Nothing
+import Standard.Base.Data.Text.Case_Sensitivity.Case_Sensitivity
+
+from project.Errors import Unsupported_Database_Operation
+
+# Specifies a set of parameters to Table.replace
+type Replace_Params
+    Value input_type (case_sensitivity : Case_Sensitivity) (only_first : Boolean)
```

Comment:
I could not figure out what type to use for `input_type`, which is the return value of Meta.type_of. It's not `Type`.

Reply:
Yeah, I don't think we have a 'higher-order' type for this. Looks fine.